opensfm/align_pdr.py

```diff
@@ -274,6 +274,13 @@ def direct_align_pdr(data, target_images=None):
     :param target_images:
     :return: reconstruction
     """
+    if data.pdr_shots_exist():
```

Comment:
this implies GPS picker must select PDR mode, correct? 
will we need to train offshore for this?
should we even run gps picker?

Reply:
For the no movement case, the gps picker would have to pass the "direct" flag in alignment_mode. Not sure if that's your question?

`pdr_shots_exists` checks whether the file exists AND not empty. If pdr_shots doesn't exist OR is empty, then we go to the no_movement branch, otherwise, we do direct alignment as before.